Documentation/CRDs/Cluster/ceph-cluster-crd.md

```diff
@@ -477,6 +488,15 @@ The following storage selection settings are specific to Ceph and do not apply t
 * `encryptedDevice`**: Encrypt OSD volumes using dmcrypt ("true" or "false"). By default this option is disabled. See [encryption](http://docs.ceph.com/docs/master/ceph-volume/lvm/encryption/) for more information on encryption in Ceph.
 * `crushRoot`: The value of the `root` CRUSH map label. The default is `default`. Generally, you should not need to change this. However, if any of your topology labels may have the value `default`, you need to change `crushRoot` to avoid conflicts, since CRUSH map values need to be unique.
 
+Allowed configurations are as follows:
+
+| block device type | limitation                                                                                            |
+|:------------------|:------------------------------------------------------------------------------------------------------|
+| disk              |                                                                                                       |
+| part              | `encryptedDevice` should be "false"                                                                   |
+| lvm               | `metadataDevice` should be "", `osdsPerDevice` should be "1", and `encryptedDevice` should be "false" |
+| crypt             |                                                                                                       |
```

Comment:
1. Can we combine these tables so we don't have duplication? 
2. I believe mpath is also supported here.

Reply:
Reflected all comments, thanks.